website/docs/docs/build/environment-variables.md

```diff
@@ -92,9 +92,27 @@ While all environment variables are encrypted at rest in dbt Cloud, dbt Cloud ha
 
 ### Special environment variables
 
-dbt Cloud has a number of pre-defined variables built in. The following environment variables are set automatically for deployment runs, and their values cannot be changed.
+dbt Cloud has a number of pre-defined variables built in. Variables are set automatically and cannot be changed.
+
+<VersionBlock firstVersion="1.6">
+
+**dbt Cloud IDE details**
+
+- `DBT_CLOUD_GIT_BRANCH`:
```

Comment:
I made the descriptions sub-bullets because all of the bullets are related to DBT_CLOUD_GIT_BRANCH, right?

Reply:
Oh I see! I was trying to follow the same structure as the other env vars.